kpi/models/asset.py

```diff
@@ -685,9 +687,9 @@ def _snapshot(self, regenerate=True):
         asset_version = self.asset_versions.first()
 
         _note = None
```

Comment:
@noliveleger, it doesn't seem like this is passed along to anything else, and [it never was](https://github.com/kobotoolbox/kpi/commit/a9c94881745b031118ec598c32116b51238f072c#diff-43986855ae1015d5975b107cf1ee9869R496) to begin with. Am I right?

Reply:
@jnm, that was my question in other comments

> Actually, I'm wondering if this condition is useful. It does nothing except assigning _note which is not used anywhere else.